docs/source/workflows/performance_workflows.rst

```diff
@@ -1,8 +1,11 @@
 Performance workflows
 =====================
 
-A performance workflow performs analysis on a module.
-Currently, the only analysis implemented is "cross validation analysis," which performs cross-validation on a predictor.
+A :class:`performance workflow <citrine.informatics.workflows.PerformanceWorkflow>` performs analysis on a module.
+On construction, a performance workflow requires a configuration object which stores all settings required to run the analysis.
+Currently, the only implemented analysis performs cross-validation on a predictor.
+Settings used to perform cross-validation are defined by a :class:`~citrine.informatics.analysis_configuration.CrossValidationAnalysisConfiguration`.
+This analysis configuration defines cross-validation parameters such as the number of folds, group-by keys (descriptor keys used to group and deduplicate candidates across folds) and :class:`others <citrine.informatics.analysis_configuration.CrossValidationAnalysisConfiguration>`.
```

Comment:
The link to the class seems redundant here, since there's also a link in the previous sentence.